### PR TITLE
Add github action for integration test

### PR DESCRIPTION
**Pull request by imeoer**
For nydus driver, use accelctl to convert OCIv1 image to nydus image,
and use nydusify checker tool to verify filesystem consistency.

For estargz driver, the filesystem consistency check needs to be done.

Close https://github.com/goharbor/acceleration-service/issues/4

Action preview: https://github.com/imeoer/acceleration-service/actions

Signed-off-by: Yan Song <imeoer@linux.alibaba.com>

### CODE REVIEW

**Comment by imeoer** [comment]
@ktock Does eStargz have any image checking tool? Is it necessary to check the filesystem consistency for original image and eStargz image in the integration test?

**Review by ktock — APPROVED** [review]
Thanks. We have [a similar test in stargz-snapshotter's CI](https://github.com/containerd/stargz-snapshotter/blob/ca0b362a713d1e7760d7a10f09f74b16550a6543/.github/workflows/tests.yml#L65-L79) but don't have standalone checking tool. We'll work on it.

File: .github/workflows/dco-check.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    branches: [ main ]
+
+jobs:
+  dco_check:
+    runs-on: ubuntu-latest
+    name: DCO Check
+    steps:
+    - name: Get PR Commits
+      id: 'get-pr-commits'
+      uses: tim-actions/get-pr-commits@ed97a21c3f83c3417e67a4733ea76887293a2c8f
```
Comment (ktock) [inline .github/workflows/dco-check.yml:14]: Any other projects using this?

**Review by ktock — CHANGES_REQUESTED** [review]

File: .github/workflows/dco-check.yml (hunk 1 above)
Comment (imeoer) [reply .github/workflows/dco-check.yml:14]: kata containers and ttrpc-rust using it:

https://github.com/kata-containers/kata-containers/blob/0570317e7b882ab87e72dde9b0539aa41eed4cc0/.github/workflows/commit-message-check.yaml#L29

https://github.com/containerd/ttrpc-rust/blob/e5a5373ca13270b608457944be231ccc71adf756/.github/workflows/sanity.yml#L26

File: .github/workflows/integration-test.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        go-version: ${{ env.GO_VERSION }}
+
+    - name: Build Accel
+      run: |
+        make install-check-tools
+        make
+
+    - name: Install Docker
+      uses: sudo-bot/action-docker-compose@latest
+
```
Comment (ktock) [inline .github/workflows/integration-test.yml:32]: Why we need to install docker here? Ubuntu runner seems to have it already: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#tools

Comment (imeoer) [reply .github/workflows/integration-test.yml]: Thanks, fixed.

**Review by ktock — APPROVED** [review]

**Pull request merged**